client/src/components/BookingCard/BookingCard.tsx

```diff
@@ -0,0 +1,70 @@
+import { Typography } from '@mui/material';
+import SettingsIcon from '@mui/icons-material/Settings';
+import AvatarDisplay from '../AvatarDisplay/AvatarDisplay';
+import { Box } from '@mui/material';
+import { Booking } from '../../interface/Booking';
+
+interface Props {
+  booking: Booking;
+  isNextBooking?: boolean;
+}
+
+export default function BookingCard({ booking, isNextBooking }: Props): JSX.Element {
+  const date = () => {
+    return (
+      '' +
+      booking.startTime.getDate() +
+      ' ' +
+      booking.startTime.toLocaleString('default', { month: 'long' }) +
+      ' ' +
+      booking.startTime.getFullYear() +
+      ', ' +
+      booking.startTime.getHours() +
+      '-' +
+      booking.endTime.getHours() +
+      ' ' +
+      (booking.endTime.getHours() >= 12 ? ' PM' : ' AM')
+    );
+  };
+
+  // The top line needs to be structured differently depending on if it is in the next-booking box or in the bottom box
+  const topLine = () => {
+    if (isNextBooking) {
```

Comment:
Nit: Using a ternary here would make the code look a bit more cleaner :)

`{isNextBooking ? ( <JSX /> ) : ( <JSX /> )}`